src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	
+	private final String PREFIX_ADDED = "+ ";
+	private final String PREFIX_DELETED = "- ";
+	
+	private Person taggedPerson;
+	private Tag tag;
+	private boolean isDeleted;
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.